src/DIRAC/Core/Utilities/ElasticSearchDB.py

```diff
@@ -240,13 +241,19 @@ def getDocTypes(self, indexName):
     return S_OK(doctype)
 
   ########################################################################
+  @ifConnected
   def exists(self, indexName):
     """
     it checks the existance of an index
 
     :param str indexName: the name of the index
     """
-    return self.client.indices.exists(indexName)
+    self.log.debug("Checking existance of index %s" % indexName)
```

Comment:
```suggestion
    sLog.debug("Checking existance of index %s" % indexName)
```
or is there a reason to use `self.log` ? Also above